swag/blockchain/blockchain_parser.py

```diff
@@ -25,7 +25,6 @@ def structure_id(o, _):
     except ValueError:
         return CagnotteId(o)
 
-
 def unstructure_id(o):
```

Comment:
Doshite?

Reply:
Eeeeh, I just did some experiment here but it was a desaster. So i removed everything except this line return xD